docs/tr1/README.md

```diff
@@ -391,6 +391,7 @@ Not all options are turned on by default. Refer to `TR1X_ConfigTool.exe` for det
 - added ability to wade, similar to TR2+
 - added Lara's exit-water-to-medium-height animation from TR2+
 - added a pause screen
+- added the ability to pause during cutscenes
```

Comment:
I think it kind of duplicates the "added a pause screen" feature – no need to mention a specialized case as a separate feature IMO

Reply:
True but we don't have pausing in FMVs yet. So it's not a universal
Pause at the moment.

Reply:
We usually use the changelog to capture version-to-version updates, whereas this document focuses more on the major differences from the original game. Since the original game didn't have a pause mode altogether, my thinking is that we don't need to include this update here.